pkg/comp-functions/functions/vshnredis/maintenance.go

```diff
@@ -9,9 +9,8 @@ import (
 )
 
 var (
-	service         = "redis"
-	maintRolename   = "crossplane:appcat:job:redis:maintenance"
-	maintSecretName = "maintenancesecret"
+	service          = "redis"
+	helmBasedService = true
 
 	policyRules = []rbacv1.PolicyRule{
```

Comment:
This is probably not necessary anymore?

Reply:
cleaned the bool variable though the service is still necessary

Reply:
I wasn't talking about either the bool nor the service but about the `policyRules`. The clusterRole is deployed via the component, right? So why do we have the same role here again?

Reply:
Ah yes could catch, let me update it :)